spec/live/webhook_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'core/spec_helper'
```

Comment:
here are the tests. Because Invocations are always empty when you create a Webhook, and given the meta-programming style of these specs (eg `it_should_be_creatable`) I wasn't able to test the empty invocation array in a timely manner, but I confirmed that it appears to be working fine in the console.